resources/views/signUp.blade.php

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+    <title>SignUp Page</title>
+    <link rel="stylesheet" href="../css/signUp.css">
+</head>
+<body>
+<h1>SignUp Page</h1>
+<div class="signUp">
+<form action="/signUp" method="POST" enctype="multipart/form-data">
+{{csrf_field()}}
+	<div class="form-group">
+	<span>Name:</span>
```

Comment:
please fix indentations

Reply:
Please follow coding standard as mention in the rules document.
https://gyazo.com/c6c73b8719ec9293cf63941db65e6eb5

html should be 4 spaces indentaion.

ex.
https://gyazo.com/9aba2b4c76900576fa58521901f85274